app.js

```diff
@@ -66,13 +68,19 @@ function displayProgress() {
 
 function addLetterListeners() {
   const letters = document.querySelectorAll('.letter-wrapper');
-  [...letters].forEach(item => item.addEventListener('click', () => letterCheck(item.id)));
+  [...letters].forEach(item => item.addEventListener('click', () => {
+    if (state.correct.includes(item.id) || state.wrong.includes(item.id)) {
```

Comment:
we could do pretty thing (as we already know spread operator):
```
if ([...state.correct, ...state.wrong].includes(item.id)) {
  return 
}
```
There will be only one loop through one array (which will break after finding a letter). Micro performance improvement.